finishedNumGame.js

```diff
@@ -0,0 +1,43 @@
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+start();
+async function start() { 
+ 
+  console.log("Comp: Let's play a game where you think of a number between 1-100 and I (Comp) try to guess it.\n");
+
+  min = 1  //assign starting minimum
+  max = 100 // assign starting maximum
+  compGuess = Math.floor((min + max) / 2) //variable to make the computer guess the median between a min/max (binary search, but not really) 
+  humanResponse = await ask('Is your number ' + compGuess + '?\n' + 'If it is... answer yes, or give me one of two hints: lower/higher.\n\n');
```

Comment:
if you define this in the top level of the while loop you will only need it in that one place, and it will automatically update based on max and min each loop through